src/main/resources/openapi/token-1.0.yaml

```diff
@@ -63,7 +65,10 @@ paths:
       - $ref: headers/okapi-tenant-required.yaml
       - $ref: headers/okapi-url-required.yaml
     post:
-      description: Sign token
+      description: |
+        Returns a signed, expiring access token and refresh token. Also returns the expiration
+        of each token in the body of the response.The access token time to live is 10 minutes and
```

Comment:
Space after dot.
Reword: The time to live is 10 minutes for the access token and one week for the refresh token.

Reply:
Done